bin/jenkins.sh

```diff
@@ -19,7 +19,11 @@ git submodule update --init
 
 rubocop=$?
 
-coverage run --source=app tests.py --xml
+createdb -O tomcat $JOB_NAME
+
+DEED_DATABASE_URI=postgres:///$JOB_NAME coverage run --source=app --omit=../*migrations* tests.py --xml
```

Comment:
I dont see a reference to this in the code, I could be wrong but I only see a reference to VERIFUDGE_DATABASE_URI in the config.py